### PR TITLE
unfuck postgres

### PR DESCRIPTION
**Pull request by Morantron**
# Wat

`bundle exec rake db:test:prepare` does not work on development, because of permission issues.

* `timeoverflow` role have SUPERUSER pawah on development
* password is removed, using `peer` authentication
* `timeoverflow_test` database should only be necessary on development actually

The following changes in `config/database.yml` are needed too: https://github.com/coopdevs/timeoverflow/pull/363

### CODE REVIEW

File: roles/database/tasks/main.yml [hunk 1]
```diff
@@ -6,17 +6,28 @@
...
     postgresql_users:
       - name: "{{ database_user }}"
         password: "{{ database_password }}"
         db: "{{ database_name }}"
+        role_attr_flags: 'SUPERUSER'
+        state: present
+      - name: "timeoverflow_test"
+        password: "{{ database_password }}"
+        db: "timeoverflow_test"
+        role_attr_flags: 'SUPERUSER'
```
Comment (Morantron) [inline roles/database/tasks/main.yml]: ~how to conditionally set this value? set it in a variable maybe?~ STFU and hostvars

File: inventory/host_vars/local.timeoverflow.org/config.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 database_name: timeoverflow_development
 rails_environment: development
+database_user_role: SUPERUSER
```
Comment (enricostano) [inline inventory/host_vars/local.timeoverflow.org/config.yml]: undefined in other stages?

Comment (Morantron) [reply inventory/host_vars/local.timeoverflow.org/config.yml]: Will it :boom:? Should I add empty `database_user_role:` in other hosts?

**Review by sauloperez — APPROVED** [review]

File: inventory/host_vars/local.timeoverflow.org/config.yml (hunk 2 above)
Comment (enricostano) [reply inventory/host_vars/local.timeoverflow.org/config.yml]: it will :boom: what are the default roles?

**Review by sauloperez — APPROVED** [review]

**Pull request merged**